filer/templates/admin/filer/delete_selected_files_confirmation.html

```diff
@@ -43,6 +43,7 @@
                 {% endif %}
                 <input type="hidden" name="action" value="delete_files_or_folders">
                 <input type="hidden" name="post" value="yes">
+                <a href="#" onclick="window.history.back(); return false;" class="button cancel-link">{% trans "No, take me back" %}</a>
```

Comment:
maybe `class='js-back-button'` and move the js out of here? I guess there can be other back buttons coming